cves/CVE-2012-3439.yml

```diff
@@ -202,8 +205,11 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: |
+      This vulnerability assumed that if a stale nonce was being given 
+      then there was no need to check authentication before reporting that it 
+      was stale. It was assumed that the request was secure by default.
```

Comment:
"noticed if a stale nonce was being given" im sorry but what does this exactly mean?"